fake-files/output-expected/linnarsson/linnarsson.images.zarr/pyramid/00/.zarray

```diff
@@ -0,0 +1,21 @@
+{
+    "chunks": [
+        2,
+        50,
+        50
+    ],
+    "compressor": {
+        "id": "zlib",
+        "level": 1
+    },
+    "dtype": "<i4",
```

Comment:
Can you figure out what is going on here?  This should be `<u4` as far as I understand

Reply:
This is a result of the data generated for the fixture (which is `<i4`). See here: 

https://github.com/hubmapconsortium/vitessce-data/blob/947bb6160dccc4afb967d05c3558a5bd7f641bff/python/create_hdf5_fixtures.py#L3-L10

Reply:
The actual linnarsson data is u16 and that is respected.

Reply:
Ah, ok.  Sounds good.